dask_expr/expr.py

```diff
@@ -674,1087 +433,3 @@ def find_operations(self, operation: type) -> Generator[Expr]:
 
             if isinstance(node, operation):
                 yield node
-
-
-class Literal(Expr):
-    """Represent a literal (known) value as an `Expr`"""
-
-    _parameters = ["value"]
-
-    def _divisions(self):
-        return (None, None)
-
-    @property
-    def _meta(self):
-        return make_meta(self.value)
-
-    def _task(self, index: int):
-        assert index == 0
-        return self.value
```

Comment:
Maybe Literal stays?  I wonder if we can remove divisions/meta and still have things work.

Reply:
I can take a look as a follow-up if ok?